tests/phpunit/bootstrap.php

```diff
@@ -4,6 +4,9 @@
  * @file
  */
 
+// Alias PHPUnit classes for backwards compatibility.
+include '../../scripts/phpunit/bootstrap.php';
```

Comment:
Is this necessary? Do internal tests break without it?

Reply:
Hmm... I _think_ this is needed for the PHP 5.6 tests. When PHP is 5.6, composer will only install PHPUnit 5, which does not have `PHPUnit\Framework\TestCase` as a valid class. This file aliases those namespaces to the old classes.

Reply:
Got it.

Reply:
Actually, Drupal core is [doing something _similar_](https://github.com/drupal/drupal/blob/8.6.x/core/tests/bootstrap.php#L197) that I just came across, because I was confused how they were going to support both versions. I guess it's just a weird in-between time with this PHP support decision.